test/src/test/java/hudson/security/HudsonPrivateSecurityRealmTest.java

```diff
@@ -67,6 +67,7 @@ public void dataCompatibilityWith1_282() throws Exception {
             // verify the sanity that the password is really used
             // this should fail
             j.createWebClient().login("bob", "bob");
+            fail();
```

Comment:
Independent I guess, but good catch.

Reply:
Yeah I saw some tests like that, if the code did not fail, the test is still passing...